src/hooks/authenticate/index.js

```diff
@@ -55,22 +55,22 @@ function authenticateHook(hook) {
     });
   }).then(response => {
     if (response.local) {
-      hook.data.userId = response.userId;
+      hook.params.user = {"id": response.userId};
```

Comment:
So, now the tests run. Can you check that your tests also run?
I believe, there might still be a problem since the authentication can not set this on the data directly. Otherwise can can not check if the user owns the resource.
We would need to add a hook in the current /resource implementation to set the userId to what comes in here.